compute_tools/src/bin/compute_ctl.rs

```diff
@@ -193,6 +193,9 @@ fn main() -> Result<()> {
     if !spec_set {
         // No spec provided, hang waiting for it.
         info!("no compute spec provided, waiting");
+
+        compute.prewarm_postgres()?;
```

Comment:
IIUC, if new config will arrive just after starting in empty mode (could happen if we take fresh compute fro mthe pool), we will wait for `compute.prewarm_postgres()` before actually doing configuration

Not sure, but maybe running prewarm in a separate thread and just logging all possible non-fatal errors is a good idea

Reply:
separate thread has the same effect. We'll get throttled.

For the sake of the experiment, I think this is fine. But yes if it works it needs to be addressed. Ideally by somehow not registering this pod as ready. But even in that case it's not super urgent. It's 100ms, only on p99.

Is it enough to add a TODO?

Reply:
added a todo.